packages/prisma-client/migrations-cli/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,45 @@
+## A migration failed in PREVIEW, how do I fix the PREVIEW database
+
+1. Open a new PR where you'll do the fix
+1. Go to https://vercel.com/webstudio-is/webstudio-designer/settings/environment-variables
+1. Set APPLY_MIGRATIONS to "true" for you PR's branch
+1. Fix the failed migration code
+1. Fix the database mannually if the migration have been applied half-way
+1. Temporarily change `ci:migrate` script to run `resolve` before applying migrations: `"ci:migrate": "migrations resolve rolled-back <name> --force && migrations migrate --force"`. Use `applied` instead of `rolled-back` if you manually applied the migration at previous step.
```

Comment:
This is clanky, would be cool if we could ssh into the Vercel build environment, and run arbitrary command there. But I didn't find a way to do that